### PR TITLE
Add RootCA Key Usage Present Lint

### PR DESCRIPTION
**Pull request by kumarde**

### CODE REVIEW

File: lints/base.go [hunk 1]
```diff
@@ -223,6 +223,7 @@ type LintReport struct {
 	WMultipleIssuer_RDN                                  ResultStruct `json:"w_multiple_issuer_rdn,omitempty"`
 	WMultipleSubject_RDN                                 ResultStruct `json:"w_multiple_subject_rdn,omitempty"`
 	ESignatureAlgorithmNotSupported                      ResultStruct `json:"e_signature_algorithm_not_supported,omitempty"`
+	ERootCaKeyUsagePresent                               ResultStruct `json:"e_root_ca_key_usage_present"`
```
Comment (zakird) [inline lints/base.go]: Probably want CA not Ca here.

Comment (kumarde) [inline lints/base.go]: The other CA lints aren't currently that either, I'll do a different PR to fix this later.

Comment (zakird) [inline lints/base.go]: sounds good.

Comment (dadrian) [inline lints/base.go]: You should check for that in `meta_lint.py`.

**Pull request merged**